app/lib/job/backend.dart

```diff
@@ -207,6 +207,7 @@ class JobBackend {
         final now = DateTime.now().toUtc();
         selected!
           ..state = JobState.processing
+          ..attemptCount = selected.attemptCount + 1
```

Comment:
This requires us to bump runtimeVersion, right?

Reply:
Yeah, we are doing that already for the current release.